devolutions-session/src/dvc/fs.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::PathBuf;
+
+pub struct ProcessTmpFile(PathBuf);
```

Comment:
question: Is it possible to hold a File instead? Currently we may need to open files using `path()`, but there is then no relation encoded into the type system and we may drop this structure while the file is still is use. I didn't read the whole code yet.

Reply:
This type is used to keep previously opened files alive until the process finishes, I'll add docs